Makefile

```diff
@@ -0,0 +1,14 @@
+# Building docker image
+VERSION = "v1.0.0"
+IM = shalsh/phenoscape-pipeline-tools
+
+build:
+	docker build -t $(IM):$(VERSION) . \
+	&& docker tag $(IM):$(VERSION) $(IM):latest
+
+run:
+	docker run --rm -ti --name phenoscape-pipeline-tools $(IM)
```

Comment:
Is there a reason you're giving the container a name? I.e., are you going to use this in a Docker Compose file or some other piece of orchestration automation?

Reply:
Also, I should add that I don't think this will even work. The Dockerfile doesn't have a default executable specified, and you're not giving a command to run here either. Did you mean to run a shell, or something else?

Reply:
> Is there a reason you're giving the container a name? I.e., are you going to use this in a Docker Compose file or some other piece of orchestration automation?

Not yet. But even then, isn't it a good idea to give a meaningful name to the container?